src/Console/PrintSchemaCommand.php

```diff
@@ -52,8 +55,10 @@ public function handle(ASTCache $cache, Filesystem $storage, SchemaBuilder $sche
         }
 
         if ($this->option('write')) {
-            $storage->put($filename, $schemaString);
-            $this->info("Wrote schema to the default file storage (usually storage/app) as {$filename}.");
+            /** @var string|null $disk */
```

Comment:
This just ignores possible errors - PHPStan is correct that the value here might not be a string or null. Instead, do an actual check and throw if the value is anything else, indicating clearly to the user what was wrong.

Reply:
I've added a check for a bool or array and returned an error to the console